internal/helpers/proxy/proxy.go

```diff
@@ -48,11 +49,16 @@ func DetermineProxyTypeForDB(conf *config.Config, dbcr *kindav1beta1.Database, i
 	}
 
 	portString := instance.Status.Info["DB_PORT"]
-	port, err := strconv.Atoi(portString)
+	port64, err := strconv.ParseInt(portString, 10, 32)
 	if err != nil {
 		logrus.Errorf("can not convert DB_PORT to int - %s", err)
 		return nil, err
 	}
+	if port64 < math.MinInt32 || port64 > math.MaxInt32 {
```

Comment:
Again, can't we use uint16 here and avoid all the checks?

Reply:
done